perf/src/sigverify.rs

```diff
@@ -162,6 +163,13 @@ pub fn count_packets_in_batches(batches: &[PacketBatch]) -> usize {
     batches.iter().map(|batch| batch.packets.len()).sum()
 }
 
+pub fn count_discard_in_batches(batches: &[PacketBatch]) -> usize {
```

Comment:
Looks like `sigverify_stage` is the only one uses this function, maybe can move it to there as private?

Reply:
seems like it would make more sense for it to live in the same module ad count packets in batches